handler/src/test/java/io/netty/handler/ssl/OpenSslEngineTest.java

```diff
@@ -565,6 +567,7 @@ public void testWrapWithDifferentSizesTLSv1(SSLEngineTestParam param) throws Exc
         testWrapWithDifferentSizes(param, SslProtocols.TLS_v1, "ECDHE-RSA-RC4-SHA");
     }
 
+    @Disabled("TLSv1.1 is not supported by BoringSSL anymore")
```

Comment:
This can be reverted once we pulled in https://github.com/netty/netty-tcnative/pull/902 and released a new version